.github/workflows/build.yaml

```diff
@@ -15,7 +15,7 @@ permissions:
 
 jobs:
   build:
-    name: build kernel #and system
+    name: build kernel and system
     runs-on: namespace-profile-arm64-8x16-2004-caching
     timeout-minutes: 20
```

Comment:
is this timeout enough without a cache?

Reply:
it should be ~40 minutes on namespace-profile-arm64-8x16-2004-caching. Updating to 40.